plugins/remsql/fdb_comm.c

```diff
@@ -688,7 +690,7 @@ void fdb_msg_clean_message(fdb_msg_t *msg)
         break;
 
     case FDB_MSG_CURSOR_OPEN:
-        if (msg->co.srcname != gbl_myhostname) {
+        if (msg->co.srcname != gbl_myhostname && msg->co.srcname != gbl_myuri) {
```

Comment:
just if (msg->co.srcname != gbl_myuri)

Reply:
I think i was crashing without msg->co.srcname != gbl_myhostname, but it's harmless, so i'm just going to leave it if I may.

Reply:
you are right. that works actually.